atecc608a_utils.c

```diff
@@ -61,3 +61,19 @@ psa_status_t atecc608a_check_config_locked()
     }
     return status;
 }
+
+psa_status_t atecc608a_genKey(uint16_t slot, uint8_t* pubkey, size_t pubkey_size)
+{
+    psa_status_t status = PSA_ERROR_GENERIC_ERROR;
+    if (pubkey != NULL && pubkey_size < ATCA_PUB_KEY_SIZE)
+    {
+       return PSA_ERROR_BUFFER_TOO_SMALL;
+    }
+
+    ASSERT_SUCCESS_PSA(atecc608a_init());
+    ASSERT_SUCCESS(atcab_genkey(slot, pubkey));
+
+exit:
+    atecc608a_deinit();
+    return status;
+}
```

Comment:
Shouldn't we add this function to the driver as the implementation for `psa_drv_se_generate_key_t`?

Reply:
Sure we can! I didn't even notice we have a place for it :)